med_io/preprocess_raw_dataset.py

```diff
@@ -56,9 +59,13 @@ def write_tfrec_and_pickle(imgs_data=None, dir_tfrec_img=None, labels_data=None,
         :return:
         """
         if imgs_data is not None:
-            write_tfrecord(imgs_data, path=dir_tfrec_img + '/'+img_tf_name+'.tfrecords')
+            write_tfrecord(imgs_data, info['validation_for_cancer'], path=dir_tfrec_img + '/'+img_tf_name+'.tfrecords')
```

Comment:
do not change default behaviour -> backward compatibility broken -> create separate function `write_tfrec_and_pickle_melanoma`

Reply:
Function "write_tfrec_and_pickle_melanoma" was added and adapted for the MELANOM dataset in preprocessing. Backward compatibility should be okay now